src/main.ts

```diff
@@ -83,6 +84,35 @@ export async function activate(
         requiredPackageIds.push('OmniSharp');
     }
 
+    const dotnetRuntimeExtensionId = 'ms-dotnettools.vscode-dotnet-runtime';
+    const requiredDotnetRuntimeExtensionVersion = '2.2.1';
```

Comment:
Too bad this can't be expressed in the package.json

Reply:
Yeah, unfortunately no versions in extension dependencies :(